Automated Smoketest: :x:
Sha: b4fdc433d8b00af842711f521c7976d170978762
Build: failed
Number of Error(s): 7
/mnt/disk1/home/vamosax/smoketest/PR-9589/HPCC-Platform/esp/bindings/SOAP/soaplib/./../../../../system/security/shared/caching.hpp:177:71: error: \xe2\x80\x98setTransactionalCacheTimeout\xe2\x80\x99 was not declared in this scope
/mnt/disk1/home/vamosax/smoketest/PR-9589/HPCC-Platform/esp/bindings/SOAP/soaplib/./../../../../system/security/shared/caching.hpp:179:56: error: \xe2\x80\x98setTransactionalCacheTimeout\xe2\x80\x99 was not declared in this scope
/mnt/disk1/home/vamosax/smoketest/PR-9589/HPCC-Platform/esp/bindings/SOAP/soaplib/./../../../../system/security/shared/caching.hpp:188:71: error: \xe2\x80\x98setTransactionalCacheTimeout\xe2\x80\x99 was not declared in this scope
/mnt/disk1/home/vamosax/smoketest/PR-9589/HPCC-Platform/esp/bindings/SOAP/soaplib/./../../../../system/security/shared/caching.hpp:190:59: error: \xe2\x80\x98setTransactionalCacheTimeout\xe2\x80\x99 was not declared in this scope
make[2]: *** [esp/bindings/SOAP/soaplib/CMakeFiles/soaplib.dir/__/__/__/platform/espprotocol.cpp.o] Error 1
make[1]: *** [esp/bindings/SOAP/soaplib/CMakeFiles/soaplib.dir/all] Error 2
make: *** [all] Error 2
Get ECLWatch build result

--- a/system/security/LdapSecurity/ldapconnection.cpp
+++ b/system/security/LdapSecurity/ldapconnection.cpp
@@ -5372,7 +5372,7 @@ private:
             prevptr = nextptr + 2;
             nextptr = strstr(prevptr, "::");
         }
-        if(prevptr != NULL && *prevptr != '\0')
+        if(*prevptr != '\0')
             ldapname.append(prevptr);
     }
 

--- a/system/security/LdapSecurity/ldapconnection.cpp
+++ b/system/security/LdapSecurity/ldapconnection.cpp
@@ -5370,8 +5370,6 @@ private:
         while(nextptr != NULL)
         {
             prevptr = nextptr + 2;
-            if (prevptr == nullptr || *prevptr == '\0')
-                break;
             nextptr = strstr(prevptr, "::");
         }
         if(prevptr != NULL && *prevptr != '\0')

